src/lib.rs

```diff
@@ -10,12 +10,11 @@
 
 //! Utilities for random number generation
 //!
-//! The key functions are `random()` and `Rng::gen()`. These are polymorphic and
-//! so can be used to generate any type supporting the [`Uniform`] distribution
-//! (i.e. `T` where `Uniform`: `Distribution<T>`). Type inference
-//! means that often a simple call to `rand::random()` or `rng.gen()` will
-//! suffice, but sometimes an annotation is required, e.g.
-//! `rand::random::<f64>()`.
+//! The key function is `Rng::gen()`. It is polymorphic and so can be used to
```

Comment:
This probably wants a bit more re-writing, but this doesn't have to be done now.

Reply:
Yes, I think we should advertise `Rng::sample` or `Distribution::sample` instead and possibly get rid of `Rng::gen`. But this is still under discussion in #293.